### ISSUE TITLE
Test failure with new ObsPy?

### ISSUE THREAD

**Issue by QuLogic** [adjtomo/pyflex#6]
There's one failure on the released version:

``` python
        # Assert the phases of the first window.
>       assert sorted([_i["phase_name"] for _i in windows[0].phase_arrivals]) == \
            [u'PKPdf', u'PKSdf', u'PKiKP', u'PP', u'SKPdf', u'SKiKP', u'pPKPdf',
             u'pPKiKP', u'sPKPdf', u'sPKiKP']
E       assert ['PKIKP', 'PK... 'SKiKP', ...] == ['PKPdf', 'PKS... 'SKiKP', ...]
E         At index 0 diff: 'PKIKP' != 'PKPdf'
E         Use -v to get the full diff

~/.local/lib/python3.4/site-packages/pyflex/tests/test_pyflex.py:135: AssertionError
```

Not sure if it's generally broken or if it's due to TauP stuff in ObsPy.

**Comment by krischer** [comment]
Should be TauP related. I'll have a look and release a new version. Anything else I should add to a new release?

**Comment by krischer** [comment]
I released 0.1.4. I hope this solves the issue.

**Comment by QuLogic** [comment]
Tests pass and the examples seem to work.
